math/twinprime.go

```diff
@@ -0,0 +1,24 @@
+// twinprime.go
+// description: Returns Twin Prime of n
+// details:
+// For any integer n, twin prime is (n + 2)
+// if and only if both n and (n + 2) both are prime
+// wikipedia: https://en.wikipedia.org/wiki/Twin_prime
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see twinprime_test.go
+
+package math
```

Comment:
There is already a [prime package](https://github.com/TheAlgorithms/Go/blob/4d9b1e48789f24959739ef03b52b26d074a98de8/math/prime/sieve.go#L6) I believe. Please move this to that package and change the name to `Twin` instead of `TwinPrime` when you do indeed move it there since `prime` is in the name.